internal/http/services/prometheus/prometheus.go

```diff
@@ -53,6 +56,13 @@ func New(m map[string]interface{}, log *zerolog.Logger) (global.Service, error)
 
 	view.RegisterExporter(pe)
 
+	// register the desired measures' views
+	view.Register(
```

Comment:
Check the result of Register as done [here](https://github.com/census-ecosystem/opencensus-go-exporter-prometheus/blob/master/example/main.go#L50) and return if there's an error.